### PR TITLE
LPS-137666 Disable component for now

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-collection-filter-date/src/main/java/com/liferay/fragment/collection/filter/date/display/context/FragmentCollectionFilterDateDisplayContext.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+public class FragmentCollectionFilterDateDisplayContext {
+
+	public FragmentCollectionFilterDateDisplayContext(
+		String configuration,
+		FragmentEntryConfigurationParser fragmentEntryConfigurationParser,
+		FragmentRendererContext fragmentRendererContext) {
+
+		_configuration = configuration;
+		_fragmentEntryConfigurationParser = fragmentEntryConfigurationParser;
+
```
Comment (ealonso) [inline modules/apps/fragment/fragment-collection-filter-date/src/main/java/com/liferay/fragment/collection/filter/date/display/context/FragmentCollectionFilterDateDisplayContext.java:39]: @victorg1991 You should removes this empty line

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**